src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let result = days * 40;
+
+  if (days >= 7) {
+    result = result - 50;
```

Comment:
Replace the literals '7' and '50' with constants to avoid hardcoding. For example, 'const LONG_TERM_RENTAL = 7;' and 'const LONG_TERM_DISCOUNT = 50;'.